packages/store/src/workspace/page.ts

```diff
@@ -507,6 +507,45 @@ export class Page extends Space<FlatBlockMap> {
     nextParentModel.propsUpdated.emit();
   }
 
+  @debug('CRUD')
+  moveBlocksToParent(blocks: BaseBlockModel[], parentId: string) {
```

Comment:
Can we reuse the existing `moveBlocks` method?

Reply:
https://github.com/toeverything/blocksuite/blob/75173db962085c38d7e676498dc755b96cfbd3ea/packages/store/src/workspace/page.ts#L483-L486

`targetModel` needs to get its parent model first.

But in this case, we already know the parent model (frameId);

Reply:
I think `moveBlocks` should be refactored based on your usage

Reply:
I try to refactor it. :)